src/main/groovy/com/netgrif/application/engine/startup/DemoRunner.groovy

```diff
@@ -3,6 +3,8 @@ package com.netgrif.application.engine.startup
 import com.netgrif.application.engine.elastic.domain.ElasticCaseRepository
 import com.netgrif.application.engine.elastic.domain.ElasticTaskRepository
 import com.netgrif.application.engine.elastic.service.interfaces.IElasticCaseService
+import com.netgrif.application.engine.petrinet.domain.PetriNet
```

Comment:
why were these imports added?

Reply:
I was testing something using DemoRunner and accidentally commited this import. It is now removed.